ZombieSharp/Statistic.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Dapper;
+using Microsoft.Extensions.Logging;
+using MySql.Data.MySqlClient;
+using Newtonsoft.Json;
+
+namespace ZombieSharp
+{
+    public partial class ZombieSharp
+    {
+        private StatsDatabase statsConfig;
+        MySqlConnection connection = null;
+
+        private void StatsOnLoad()
+        {
+            var statsConfigPath = Path.Combine(ModuleDirectory, $"../../configs/zombiesharp/database.json");
+            statsConfig = JsonConvert.DeserializeObject<StatsDatabase>(File.ReadAllText(statsConfigPath));
+            connection = StatsConnect();
+        }
+
+        private MySqlConnection StatsConnect()
+        {
+            var query = $"Server={statsConfig.hostname}; database={statsConfig.database}; UID={statsConfig.username}; password={statsConfig.password}";
+            var conn = new MySqlConnection(query);
+
+            if(conn == null)
+            {
+                Logger.LogError("[Stats] Connection is null.");
+                return null;
+            }
+
+            conn.Open();
+
+            return conn;
+        }
+
+        private async Task StatsSetData(CCSPlayerController client, int damage = 0, int kill = 0, int infect = 0)
```

Comment:
It's better to move this method to some service, like `PlayerStatsService`. Do not include everything in one class. Avoid god objects https://en.wikipedia.org/wiki/God_object